.github/workflows/main.yml

```diff
@@ -1,9 +1,6 @@
 name: Pipeline
 
 on:
-    pull_request:
```

Comment:
We should keep the checks during pull requests (PR) to ensure the tests and quality checks are done when a PR is opened.

Reply:
Done!